daemon/common.go

```diff
@@ -35,10 +35,10 @@ func InstallDaemon() error {
 
 // Restart - restarts a system daemon
 func Restart() error {
```

Comment:
I am thinking for these sorts of functions, it would be better to have daemon/common_windows.go, common_linux.go, as it would have less code in each artifact then, could be separate PR though

Reply:
GRA-760